module/input/SensorFilter/src/SensorFilter.cpp

```diff
@@ -54,13 +54,10 @@ namespace module::input {
     using utility::nusight::graph;
     using utility::support::Expression;
 
-    std::string makeErrorString(const std::string& src, uint errorCode) {
+    [[nodiscard]] std::string makeErrorString(const std::string& src, uint errorCode) {
         std::stringstream s;
 
-        s << "Error on ";
-        s << src;
-        s << ":";
-
+        s << "Error on " << src << ":";
 
         if (errorCode & RawSensors::Error::INPUT_VOLTAGE) {
```

Comment:
I think you should change the checks to this sort of format
```suggestion
        if (errorCode & RawSensors::Error::INPUT_VOLTAGE == RawSensors::Error::INPUT_VOLTAGE) {
```
or this if you prefer
```suggestion
        if (errorCode & RawSensors::Error::INPUT_VOLTAGE != 0) {
```

Reply:
Dont forget the rest of them ☺ (also might be a good idea to verify that it works)

Reply:
Do you mean for all of those if statements with similar things to do the same? this change has added a warning so I will still have to clear that up as well

Reply:
Could we merge this PR and I will make another PR for these changes and other small changes like these. Could you also check the servo error check as well (lines 217 - 239)

Reply:
I think this would be better in a follow up. This seems a little scope-creep-y to me, given that these lines weren't changed and this wasn't the intention of the PR